From 9b6890332084e49dc7c9e64013704b9c14eb0678 Mon Sep 17 00:00:00 2001
From: Marco Contenti <marco.c@variscite.com>
Date: Tue, 9 Nov 2021 15:40:14 +0100
Subject: [PATCH 5/5] ar1335_af: add missing compatible specification

Signed-off-by: Marco Contenti <marco.c@variscite.com>
---
 drivers/media/platform/imx8/ar1335_af.c | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/drivers/media/platform/imx8/ar1335_af.c b/drivers/media/platform/imx8/ar1335_af.c
index 84fc7ba7abdb..6714c6816696 100644
--- a/drivers/media/platform/imx8/ar1335_af.c
+++ b/drivers/media/platform/imx8/ar1335_af.c
@@ -3507,6 +3507,16 @@ static int ar1335_remove(struct i2c_client *client)
 	return 0;
 }
 
+#ifdef CONFIG_OF
+static const struct of_device_id ar1335_af_of_match[] = {
+	{ .compatible = "ar1335_af",
+	},
+	{ /* sentinel */ }
+};
+
+MODULE_DEVICE_TABLE(of, ar1335_af_of_match);
+#endif
+
 static const struct i2c_device_id ar1335_af_id[] = {
 	{"ar1335_af", 0},
 	{}
-- 
2.33.0

